### DISCUSSION TITLE
Any auth examples?

### DISCUSSION THREAD

**Discussion by bbigras** [freshframework/fresh#284] [Q&A]
I would like to test fresh with auth.

Anything I need to do differently with fresh vs normal react?

Something like next-auth would be nice too.

**Comment by seanaye** [comment]
AFAIK, fresh is pretty much server only which means that auth would have to be done with session cookies

**Reply by sophek** [reply]
I would love to see an example to integrate sessions from other libraries... like https://deno.land/x/sessions@v1.4.0

**Comment by thesiti92** [comment]
something about auth in the docs/examples would be very handy. I can try and contribute it if need be

**Comment by excessivecoding** [comment]
You might find some guidance on some Deno Deploy's showcase projects which run on Fresh: https://github.com/denoland/showcase_chat
